### DISCUSSION TITLE
AppShell: non-sticky full-width footer

### DISCUSSION THREAD

**Discussion by officialankan** [skeletonlabs/skeleton#2471] [Q&A] [answered]
Hi!

I am constructing a dashboard like page using an AppShell with a header, left sidebar and footer. The footer is sticky which is expected according to the app shell documentation. To make it non-sticky, I get that I can turn it into a **page**Footer - but that makes it not fill the entire width.

Is there a way to make the non-sticky **page**Footer fill the full width of the screen (like the ordinary footer)? Another solution would be a way to make the footer non-sticky...

Thanks!

![bild](https://github.com/skeletonlabs/skeleton/assets/13569761/ed4b5108-f40e-4284-af89-7649d5c95b53)

**Comment by Sarenor** [comment, marked as answer]
Not with the AppShell, but if that is what you want, especially when the sidebarLeft always needs to be visible, you can easily build a similar setup using GridAreas.
The AppShell is there as a help to get started, when it no longer does what you want it to, it's time to ditch it and implement a custom layout

**Reply by officialankan** [reply]
This was my guess but I figured I had to ask to see if there was a quick fix. Thanks for the quick reply!
